install-controltower.md

```diff
@@ -103,7 +112,7 @@ Some configuration elements need to be updated when using ControlTower
 - global-config.yaml 
   * Update `managementAccountAccessRole` value to `AWSControlTowerExecution`
   * Update `controlTower` to `enable: true`
-  * Uncomment the `landingZone` block
+  * Uncomment the `landingZone` block (only if it is a new Organization with zero accounts other than the Management Account)
```

Comment:
Per [Auto-deploy AWS Control Tower by the solution (recommended)
](https://docs.aws.amazon.com/solutions/latest/landing-zone-accelerator-on-aws/prerequisites.html#for-aws-control-tower-based-installation):

> There are no AWS services enabled for AWS Organizations.
> There are no organization units created in the AWS Organizations.
> The **only AWS account in the AWS Organization** is the management account.